{
  Throwable exception=null;
  try {
    super.delete(path);
  }
 catch (  Throwable t) {
    exception=t;
  }
 finally {
    sop("delete: " + path(path),exception);
  }
}
